scheduler/src/cook/kubernetes/api.clj

```diff
@@ -1204,36 +1204,62 @@
                           (t/plus (t/seconds unschedulable-seconds))
                           (t/before? (t/now)))))))))
 
-(defn pod-containers-not-initialized?
-  "Returns true if the given pod status has an Initialized condition with status
-  False and reason ContainersNotInitialized, and the last transition was more than
-  pod-condition-containers-not-initialized-seconds seconds ago"
-  [pod-name ^V1PodStatus pod-status]
-  (let [{:keys [pod-condition-containers-not-initialized-seconds]} (config/kubernetes)
-        ^V1PodCondition pod-condition
+(defn pod-has-terminal-condition?
```

Comment:
maybe "stuck" or "unready" instead of "terminal"?
terminal sounds like pod finished running

Reply:
Good call, renamed to "stuck"